test/parser.jl

```diff
@@ -86,3 +86,14 @@ macro test999_str(args...); args; end
 
 # issue #10985
 @test expand(:(f(::Int...) = 1)).head == :method
+
+# issue #10910
+@test parse(":(using A)") == Expr(:quote, :(using A))
```

Comment:
This still doesn't test the parser. If `using A` were parsed as the number 42, this test would still pass.